backend/services/theme_service.py

```
@@ -77,97 +77,342 @@ def _create_smart_person_background_mask(self, image_bytes: bytes, image_width:
+            original_image_bytes: Original Swamiji photo for reference
+            mask_bytes: Mask used for inpainting (to identify preserved regions)
+            image_width: Image width
+            image_height: Image height
+            
+        Returns:
+            bytes: Color-harmonized image with natural face-body blending
+        """
+        try:
+            # Load images
+            generated_image = Image.open(io.BytesIO(generated_image_bytes)).convert('RGB')
+            original_image = Image.open(io.BytesIO(original_image_bytes)).convert('RGB')
```

Ruff (0.12.2)

Local variable `original_image` is assigned to but never used

Remove assignment to unused variable `original_image`

(F841)